sdk/rust/oak/src/lib.rs

```diff
@@ -67,11 +68,11 @@ pub mod proto {
 
 // Build a chunk of memory that is suitable for passing to oak_abi::wait_on_channels,
 // holding the given collection of channel handles.
-fn new_handle_space(handles: &[ReadHandle]) -> Vec<u8> {
+fn new_handle_space<R: Borrow<ReadHandle>>(handles: &[R]) -> Vec<u8> {
```

Comment:
Is this just temporary until we modify all the call places to use actual borrows, or do you think we will want to keep it after that? I think it does not look very nice, so I hope we can get rid of it at some point (perhaps worth a TODO + issue?)

Reply:
Once linear handles are fully implemented and the default, then this can be simplified again. I'll add a TODO and link to the linear handles issue.